### ISSUE TITLE
Option+Arrow causes hang in SQL Editor

### ISSUE THREAD

**Issue by stephenreay** [TablePlus/TablePlus#1927]
1. Which driver are you using and version of it (Ex: PostgreSQL 10.0):
MySQL 5.x

2. Which TablePlus build number are you using (the number on the welcome screen, Ex: build 81):
315

3. The steps to reproduce this issue:
- Connect to server, open SQL editor pane.
- Try to move the cursor forward/backward by word using Option + Left/Right Arrow key
- TablePlus hangs.

**Comment by huyphams** [comment]
Hi @stephenreay thank you for letting me know. Can I have the query string? I couldn't reproduce the issue.

**Comment by stephenreay** [comment]
Oh wow that was quick. 

Ok and of course now *I* can't reproduce it either (it happened reliably, about 5 or 6 times with the same query, copied below). But in trying to work out why it suddenly doesn't do it, I opened the query in question with a hex editor and noticed that it somehow has the ASCII enquiry character (HEX `05`) in it, which I can't really work out, because I typed the query, it's not something copied from somewhere else.


```SQL
select purchases_datetime from purchases where stats_avg_amount is null order by purchases_id desc limit 0,20 
```
